android/app/build.gradle

```diff
@@ -81,6 +81,9 @@ android {
                     signingConfig signingConfigs.release 
                 }
             }
+            ndk {
+                abiFilters 'armeabi-v7a','arm64-v8a','x86_64'
```

Comment:
If this is required on andriod, we probably need to test this on iOS before releasing

Reply:
That is true, I didn't think of that.